docs/pages/guides/nextjs-starter-kit.mdx

```diff
@@ -30,104 +30,58 @@ The Next.js Starter Kit includes the following
-page), components, which you can think of as multiplayer implementations
-(cursors, badges, etc), pages, which render document level experiences and both
-server side and client side methods which contain the logic to create and modify
-documents.
+You can get started by running the following command:
 
-<br />
-<details>
-  <summary>More info about the structure</summary>
-  <br />
-  <dl>
-    <dt>`/components/`</dt>
-    <dd>Components used in the app.</dd>
-    <dt>`/data/`</dt>
-    <dd>A demo database.</dd>
-    <dt>`/layouts/`</dt>
-    <dd>Page layout components.</dd>
-    <dt>`/lib/client/`</dt>
-    <dd>Client functions used for accessing documents, database, SWR hooks.</dd>
-    <dt>`/lib/server/`</dt>
-    <dd>
-      Server functions used for accessing documents, database, liveblocks.
-    </dd>
-    <dt>`/pages/`</dt>
-    <dd>Next.js pages.</dd>
-    <dt>`/primitives/`</dt>
-    <dd>Generic reusable components.</dd>
-    <dt>`/types/`</dt>
-    <dd>All reusable TypeScript types.</dd>
-    <dt>`/liveblocks.config.ts`</dt>
-    <dd>Liveblocks config file.</dd>
-    <dt>`/liveblocks.server.config.ts`</dt>
-    <dd>Server-side Liveblocks config file.</dd>
-  </dl>
-</details>
-
-### Liveblocks configuration
-
-If you already have a Liveblocks Development account, you may skip to step 4.
+```shell
+npx create-liveblocks-app@latest --next
+```
 
-1. Create an account at [liveblocks.io](https://liveblocks.io])
-2. Navigate to the [dashboard](https://liveblocks.io/dashboard) and click "Add
-   project..."
-3. Create a new project with "Development" as the environment.
-4. Go to "API keys" on the side menu, reveal your secret key, and copy it
+This will run an installer that allows you to download, configure, and deploy, your project:
 
 <Figure>
   <video autoPlay loop muted playsInline>
     <source
-      src="/images/docs/next-js-starter-kit/create-api-key.mp4"
+      src="/images/docs/next-js-starter-kit/create-liveblocks-app.mp4"
       type="video/mp4"
     />
   </video>
 </Figure>
 
-5. From the command line, get the starter kit by running Create Liveblocks App. Follow through the instructions, selecting your project name, chosen authentication method, and install the app:
+Give your project a name, and select the [authentication](#authentication) method you’d like to use in your app.
+If you are trying to generate a proof of concept quickly, opting to use demo authentication may be a great option—you can still add other authentication providers later on.
 
-```shell
-npx create-liveblocks-app --next
-```
+#### Deploy with Vercel Integration
+If you would like to set up CI/CD with your application, we've made that process straightforward for you as well.  When prompted, you can select "deploy to Vercel" to enable building and deploying the starter kit. The Vercel Integration will open in a new browser window for you to complete the process of adding your repository.
```

Comment:
```suggestion
If you would like to set up CI/CD with your application, we’ve made that process straightforward for you as well. When prompted, you can select "deploy to Vercel" to enable building and deploying the starter kit. The Vercel Integration will open in a new browser window for you to complete the process of adding your repository.
```

Double spaces @taylorpwhipp and curly apostrophe. Can you make they're all checked?

Reply:
I fixed them to approve the PR

Reply:
Thanks!